src/main/java/core/mate/academy/service/TruckProducer.java

```diff
@@ -0,0 +1,17 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Machine;
+import core.mate.academy.model.Truck;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer {
+    @Override
+    public List<Machine> get() {
+        List<Machine> resultTruck = new ArrayList<>();
+        resultTruck.add(new Truck("tr1","t1",10));
```

Comment:
add space after each comma. Use IDEA's auto formatter before submitting (Ctrl Alt L for windows)